### ISSUE TITLE
item variable is usable within task name

### ISSUE THREAD

**Issue by csutherl** [ansible-middleware/jws-ansible-playbook#82]
When running the collection, you see the following output for the remove webapps task:

```
TASK [middleware_automation.jws.jws : Remove app: {{ item }}] ********************************************
changed: [localhost] => (item=docs)  
changed: [localhost] => (item=ROOT)  
ok: [localhost] => (item=examples)  
```

Note that the variable `item` isn't interpolated, which seems to be due to the fact that the scope of the variable is inside the task. We should either change the task name to use `{{ tomcat.apps.to_remove }}` to print the list of apps to remove, or change it to something like "Remove applications". We don't need to print for each iteration since Ansible does that for us here noting what changed or not.

**Comment by rpelisse** [comment]
Yeah... Not my proudest thing, I should have removed {{ item }} a long time ago. 

As the loop names each files removed, I suggest we just move "Removes apps:" for the name.

**Comment by rpelisse** [comment]
Fixed by [PR86](https://github.com/ansible-middleware/jws-ansible-playbook/pull/86)
